sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala

```diff
@@ -758,40 +719,15 @@ class DDLParserSuite extends AnalysisTest with SharedSQLContext {
         "SET FILEFORMAT PARQUET")
   }
 
-  test("alter table: set location") {
-    val sql1 = "ALTER TABLE table_name SET LOCATION 'new location'"
+  test("alter table: set partition location") {
     val sql2 = "ALTER TABLE table_name PARTITION (dt='2008-08-08', country='us') " +
       "SET LOCATION 'new location'"
-    val parsed1 = parser.parsePlan(sql1)
     val parsed2 = parser.parsePlan(sql2)
     val tableIdent = TableIdentifier("table_name", None)
-    val expected1 = AlterTableSetLocationCommand(
-      tableIdent,
-      None,
-      "new location")
     val expected2 = AlterTableSetLocationCommand(
       tableIdent,
       Some(Map("dt" -> "2008-08-08", "country" -> "us")),
       "new location")
-    comparePlans(parsed1, expected1)
-    comparePlans(parsed2, expected2)
-  }
-
-  test("alter table: change column name/type/comment") {
```

Comment:
This test should be preserved in `PlanResolutionSuite`?

Reply:
Good catch, looks like I missed this one.

Reply:
The parser didn't change for this variant of `CHANGE COLUMN`, so I'm adding it back. Looks like I just removed it accidentally.